src/modules/CRUD/components/Details/ExportGeomFiles/ExportGeomFiles.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Api from '@terralego/core/modules/Api';
+import { useTranslation } from 'react-i18next';
+import { Classes, MenuDivider, MenuItem } from '@blueprintjs/core';
+
+import { sanitizeCustomEndpoint } from '../../../services/utils';
+
+const ExportGeomFiles = ({ files }) => {
+  const list = Object.entries(files);
+  const { t } = useTranslation();
+
+  const count = list.length;
+
+  if (!count) {
+    return null;
+  }
+
+  const handleClick = async event => {
+    event.preventDefault();
+    const href = event.target.closest('a').getAttribute('href');
```

Comment:
Not a fan of doing that. Can you set a state with the file and show the element if the state is not null instead ?

Reply:
`Element.closest()` is not supported by IE at all. *(I do not know IE support status of terra-admin)*

![image](https://user-images.githubusercontent.com/1846633/95179651-38ebeb80-07c1-11eb-95bf-d474f9ca7e57.png)

Reply:
It's not a compat issue for me but more a philosophy question. Should we really do DOM manipulation through js API in a react code ?

Reply:
Concerning @mabhub issue, i thought `'react-app-polyfill/ie11'` worked with it, but apparently nothing about traversing DOM. So i might replace `event.target.closest('a')` by `event.currentTarget` (which is finally better).

@jrmi would you prefer the pass the href directly as parameter in the `handleClick`?
```jsx
      {list.map(([name, href]) => (
        <MenuItem
          className={Classes.MINIMAL}
          icon="document"
          text={name}
          key={name}
          href={href}
          onClick={handleClick(href)}
        />
      ))}
```
and 
```js

  const handleClick = href => async event => {
    event.preventDefault();
    const data = await Api.request(sanitizeCustomEndpoint(href), { responseType: 'text' });
    const file = new Blob([data], { type: 'text/plain' });

    const [fileName] = href.split('/').slice(-1);
    const link = document.createElement('a');
    link.download = fileName;
    link.href = global.URL.createObjectURL(file);
    link.click();
  };
```